src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -82,6 +84,30 @@ protected static void addOutputFieldsFormatConfigGroup(final ConfigDef configDef
             FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG,
             FixedSetRecommender.ofSupportedValues(OutputFieldEncodingType.names())
         );
+
+        addFormatTypeConfig(configDef, formatGroupCounter);
+    }
+
+    protected static void addFormatTypeConfig(final ConfigDef configDef,
+                                              final int formatGroupCounter) {
+        configDef.define(
+            FORMAT_OUTPUT_TYPE_CONFIG,
+            ConfigDef.Type.STRING,
+            FormatType.CSV,
+            new OutputTypeValidator(),
+            ConfigDef.Importance.MEDIUM,
+            "The format type of output content"
+                + "The supported values are: " + FormatType.SUPPORTED_FORMAT_TYPES + ".",
```

Comment:
This string will look bad when actually concated and printed :)

Reply:
- [ ] Fix formatting